distribution/lib/Standard/Base/0.0.0-dev/src/System/Process/Exit_Code.enso

```diff
@@ -30,9 +30,9 @@ type Exit_Code
        > Example
          Convert a success code to a corresponding number.
 
-             import Standard.Base.System.Process.Exit_Code
+             import Standard.Base.System.Process.Exit_Code.Exit_Code
 
-             example_to_number = Exit_Code.Exit_Success.to_number
+             example_to_number = Exit_Code.Success.to_number
```

Comment:
Makes me remember that we used to have plans to have an automatic compilation of Examples.

I think it is something worth considering soon-ish so that we can assure that the examples provided in the docs are correct.

IMO likely this may not take much more time than manually making sure all of them work - and before an official release we definitely should make sure the examples are correct - giving incorrect examples to the users is going to be extremely annoying for them. And given so many refactors I'm almost sure there are other examples which are broken, however careful we would be it's hard to keep track of all.

Reply:
agree need to validate examples - yes picking them out and building them automagically seems right to me.